src/core/call/filter_fusion.h

```diff
@@ -536,16 +626,44 @@ auto ExecuteCombinedWithChannelAccess(Call* call, Derived* channel,
       typename FilterMethods::Methods(), typename FilterMethods::Idxs());
 }
 
-#define GRPC_FUSE_METHOD(name, type, forward)                                 \
+// Combine the result of a series of OnFinalize filter methods into a single
+// method.
+template <typename Call, typename Derived, typename T, typename Filter0,
+          typename... Filters, auto filter_method_0, auto... filter_methods,
+          size_t I0, size_t... Is>
+auto ExecuteCombinedOnFinalizeWithChannelAccess(
+    Call* call, Derived* channel, T* call_final_info,
+    Typelist<Filter0, Filters...>,
+    Valuelist<filter_method_0, filter_methods...>,
+    std::index_sequence<I0, Is...>) {
+  return TrySeq(AdaptMethod<T, decltype(filter_method_0), filter_method_0>(
```

Comment:
OnFinalize is guaranteed always synchronous, so we can omit the TrySeq here and get simpler code.

Reply:
Fixed